content/en/docs/configuring-sdk/_index.md

```diff
@@ -67,19 +67,250 @@ export AWS_REGION=us-west-2
+* `AWS_SESSION_TOKEN` (optional)
+
+The following examples show how you configure the environment variables.
+
+#### Linux, OS X, or Unix
+```bash
+$ export AWS_ACCESS_KEY_ID=YOUR_AKID
+$ export AWS_SECRET_ACCESS_KEY=YOUR_SECRET_KEY
+$ export AWS_SESSION_TOKEN=TOKEN
+```
+
+#### Windows
+
+```batch
+> set AWS_ACCESS_KEY_ID=YOUR_AKID
+> set AWS_SECRET_ACCESS_KEY=YOUR_SECRET_KEY
+> set AWS_SESSION_TOKEN=TOKEN
+```
+
+### Specify Credentials Programmatically
+`config.LoadDefaultConfig` allows you to provide an explicit
+[aws.CredentialProvider]({{< apiref "aws#CredentialsProvider" >}}) when loading the shared configuration sources.
+To pass an explicity credential provider when loading shared configuration use
+[config.WithCredentialsProvider]({{< apiref "config#WithCredentialsProvider" >}}). For example, if `customProvider`
+references an instance of `aws.CredentialProvider` implementation, it can be passed during configuration loading
+like so:
+
+```go
+cfg, err := config.LoadDefaultConfig(config.WithCredentialsProvider(customProvider))
+```
+
+A credential provider passed in this manner will be exclusivly be used by the SDK, and no other credentials will be 
+loaded from other sources.
+
+{{% pageinfo color="info" %}}
+All credential providers passed to or returned by `LoadDefaultConfig` will be wrapped in a
+[CredentialsCache]({{< apiref "aws#CredentialsCache" >}}) automatically. This allows for caching and concurrency safe 
+credential access. If you explicitly configure a provider on `aws.Config` directly you will need to wrap the provider
+with this type explicitly.
+{{% /pageinfo %}}
+
+#### Static Credentials
+
+You can hard-code credentials in your application by using the [credentials.StaticCredentialsProvider]({{< apiref "credentials#StaticCredentialsProvider" >}})
+credential provider to explicitly set the access keys to be used. For example:
+
+```go
+cfg, err := config.LoadDefaultConfig(
+	config.WithCredentialsProvider(aws.StaticCredentialsProvider("AKID", "SECRET_KEY", "TOKEN")),
+)
+```
+
+{{% pageinfo color="warning" %}}
+Do not embed credentials inside an application. Use this
+method only for testing purposes.
+{{% /pageinfo %}}
+
+#### Other Credentials Providers
+
+The SDK provides other methods for retrieving credentials in the
+[credentials]({{< apiref credentials >}}) module. For example, you can retrieve temporary security credentials from {{%
+alias service=STSlong %}} or credentials from encrypted storage.
+
+**Available Credential Providers**:
+
+* [ec2rolecreds]({{< apiref "credentials/ec2rolecreds" >}}) &ndash; Retrieve Credentials from {{< alias service=EC2 >}}
+  Instances Roles via {{< alias service=EC2 >}} IMDS.
+
+* [endpointcreds]({{< apiref "credentials/endpointcreds" >}}) &ndash; Retrieve Credentials from an arbitrary HTTP
+  endpoint.
+
+* [processcreds]({{< apiref "credentials/processcreds" >}}) &ndash; Retrieve Credentials from an external process that
+  will be invoked via the host environments shell.
```

Comment:
"will be invoked via the host environments shell" -> "the host environment's shell invokes"